core/src/main/scala/org/http4s/Method.scala

```diff
@@ -54,54 +53,55 @@ object Method extends MethodInstances {
 
   import Semantics._
 
-  // Lookups will usually be on fromString, so we store it wrapped in a Right
-  private val registry = TrieMap[String, Right[Nothing, Method]]()
+  // TODO: find out the rest of the body permissions. http://www.iana.org/assignments/http-methods/http-methods.xhtml#methods
+  val ACL: Method = new Method("ACL") with Idempotent
+  val `BASELINE-CONTROL`: Method = new Method("BASELINE-CONTROL") with Idempotent
+  val BIND: Method = new Method("BIND") with Idempotent
+  val CHECKIN: Method = new Method("CHECKIN") with Idempotent
+  val CHECKOUT: Method = new Method("CHECKOUT") with Idempotent
+  val CONNECT: Method = new Method("CONNECT") with Default with NoBody
+  val COPY: Method = new Method("COPY") with Idempotent
+  val DELETE: Method = new Method("DELETE") with Idempotent with NoBody
+  val GET: Method = new Method("GET") with Safe with NoBody
+  val HEAD: Method = new Method("HEAD") with Safe with NoBody
+  val LABEL: Method = new Method("LABEL") with Idempotent with PermitsBody
+  val LINK: Method = new Method("LINK") with Idempotent
+  val LOCK: Method = new Method("LOCK") with Default
+  val MERGE: Method = new Method("MERGE") with Idempotent
+  val MKACTIVITY: Method = new Method("MKACTIVITY") with Idempotent
+  val MKCALENDAR: Method = new Method("MKCALENDAR") with Idempotent
+  val MKCOL: Method = new Method("MKCOL") with Idempotent
+  val MKREDIRECTREF: Method = new Method("MKREDIRECTREF") with Idempotent
+  val MKWORKSPACE: Method = new Method("MKWORKSPACE") with Idempotent
+  val MOVE: Method = new Method("MOVE") with Idempotent
+  val OPTIONS: Method = new Method("OPTIONS") with Safe with PermitsBody
+  val ORDERPATCH: Method = new Method("ORDERPATCH") with Idempotent
+  val PATCH: Method = new Method("PATCH") with Default with PermitsBody
+  val POST: Method = new Method("POST") with Default with PermitsBody
```

Comment:
On the list of methods there is a `PRI` method not present here. It seems to be an http2 method. Any reason it wasn't included here?